### PR TITLE
Ds ethiop11new

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Final changes.

### CODE REVIEW

File: DayrSuryan/DSEthiop11.xml [hunk 1]
```diff
@@ -452,9 +452,16 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   </objectDesc>
 
                   <handDesc>
-                     <handNote xml:id="h1" script="Ethiopic">
-                        <seg type="script"/>
-                     </handNote>
+                    <handNote script="Ethiopic" xml:id="h1" corresp="#p1.2 #p1.3 #a4">
+          <seg type="script">First half of the 20th-century script</seg>
+          <seg type="ink">Black, red</seg>
+          <seg type="rubrication"></seg>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop11.xml:458]: If nothing will be added in rubrication, the empty element can be deleted

File: DayrSuryan/DSEthiop11.xml [hunk 2]
```diff
@@ -489,7 +496,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   <bindingDesc>
                      <binding xml:id="binding">
                         <decoNote xml:id="b1">Non-Ethiopian binding, cardboard, reddish leather, endbands. One imprinted border, one imprinted
-                        white cross in the centre on each board, يوسف نجده الحبشي imprinted on the spine.</decoNote>
+                        white cross in the centre on each board, the name of the owner
+                        يوسف نجده الحبشي is imprinted on the spine.</decoNote>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop11.xml:500]: The name should be marked up with `<foreign xml:lang="ar">`

File: DayrSuryan/DSEthiop11.xml [hunk 3]
```diff
@@ -505,6 +513,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                    the Evangelist, during the tenure of Coptic Patriarch Cyril V (1874-1927).</origDate>
                    The manuscript was copied in Egypt, by the scribe
                    <persName ref="PRS11958WaldaM" role="scribe">Walda Māryām</persName>.
+
+                  The owner or commissioner of the manuscript might be the person whose name is imprinted on the spine,
+                  cp. <forename xml:lang="ar"> يوسف نجده الحبشي </forename>, that can be interprerted as
```
Comment (thea-m) [inline DayrSuryan/DSEthiop11.xml:518]: interpreted

File: DayrSuryan/DSEthiop11.xml [hunk 4]
```diff
@@ -505,6 +513,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                  the Arabic translation of the name <persName ref="PRS12972Yosef">Yosif ʾAggazaw "the Abyssinian"</persName>.
```
Comment (thea-m) [inline DayrSuryan/DSEthiop11.xml:519]: here also Yosef, I think. And it sounds maybe a bit too certain to me? Maybe better something "...translation of an Ethiopian name such as...."?

File: DayrSuryan/DSEthiop11.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop11.xml:458]: I prefer not to delete. Someone will want to fill it out in the future.

File: DayrSuryan/DSEthiop11.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop11.xml:500]: Done

File: DayrSuryan/DSEthiop11.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop11.xml:518]: Ok

File: DayrSuryan/DSEthiop11.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop11.xml:519]: I mitigated it a bit.

**Pull request merged**